be/src/database/migrations/1619889510529-items.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
What is this migration?

Reply:
This seems to be a migration that creates a table named 'items'. That is the back end's migration. This is out of the scope of this PR. This PR was meant to remove the example data created by the database itself (unrelated to the back end). This PR is not modifying the back end's migrations.

Reply:
I know what this migration is. I'm asking why is it in this pr?

Reply:
> I know what this migration is. I'm asking why is it in this pr?

It seems that i have accidentally commited it. I pulled some other branches for development. I will remove it from this PR history.

Reply:
Accidentally added file: `be/src/database/migrations/1619889510529-items.ts` removed from history with force pushed commit: c0560007ad374e15eb16b32f6071dff1022371c2.